storage/kvstore.go

```diff
@@ -504,3 +504,23 @@ func revBytesRange(rev revision) (start, end []byte) {
 
 	return start, end
 }
+
+// NewKV is only used by tools/storage-bench
```

Comment:
Just expose `NewStore(path string, batchInterval time.Duration, batchLimit int)` and change current callers of `newStore` to `newDefaultStore(path)` or so